homeassistant/components/kostal_plenticore/sensor.py

```diff
@@ -232,7 +233,7 @@ class PlenticoreSensorEntityDescription(
         key="Cycles",
         name="Battery Cycles",
         icon="mdi:recycle",
-        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
For reviewer: This entity counts the number of full cycles of the battery. Imagine like a energy counter but with different units.